colibri/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+cargo-features = ["edition2021"]
```

Comment:
This gives a warning that is not needed anymore if you check the CI

Reply:
Ok, so this was only needed because we are using a really old compiler version for the docker image. Let's update the docker image to use the stable toolchain instead.